src/terminal/parser/stateMachine.cpp

```diff
@@ -385,7 +385,7 @@ static constexpr bool _isActionableFromGround(const wchar_t wch) noexcept
 // - wch - Character to dispatch.
 // Return Value:
 // - <none>
-void StateMachine::_ActionExecute(const wchar_t wch) noexcept

```

Comment:
I don't think so. If an exception is getting this far, it's likely because we want it to fall through to the next level, otherwise it would typically have been handled in the `_SafeExecute` method.